node/src/GlideClient.ts

```diff
@@ -657,6 +660,50 @@ export class GlideClient extends BaseClient {
         return this.createWritePromise(createFunctionKill());
     }
 
+    /**
+     * Returns the serialized payload of all loaded libraries.
+     *
+     * @see {@link https://valkey.io/commands/function-dump/|valkey.io} for details.
+     * @remarks Since Valkey version 7.0.0.
+     *
+     * @returns The serialized payload of all loaded libraries.
+     *
+     * @example
+     * ```typescript
+     * const data = await client.functionDump();
+     * // data can be used to restore loaded functions on any Valkey instance
+     * ```
+     */
+    public async functionDump(): Promise<Buffer> {
+        return this.createWritePromise(createFunctionDump(), {
+            decoder: Decoder.Bytes,
+        });
+    }
+
+    /**
+     * Restores libraries from the serialized payload returned by {@link functionDump}.
+     *
+     * @see {@link https://valkey.io/commands/function-restore/|valkey.io} for details.
+     * @remarks Since Valkey version 7.0.0.
+     *
+     * @param payload - The serialized data from {@link functionDump}.
+     * @param policy - (Optional) A policy for handling existing libraries.
```

Comment:
Should this say what happens if the policy isn't specified?

Reply:
Nice finding, added.